joomla/joomla-cms
# Added the ordered list feature

Pull Request for Issue #39532 .

### Summary of Changes
Added the ordered list feature by sorting them


### Testing Instructions
Make sure you have sufficient category items. Now when you create an article you can see the category is in order.


### Actual result BEFORE applying this Pull Request
![image](https://github.com/joomla/joomla-cms/assets/11453784/372a48ef-3e92-4498-b4af-d0eda2e09ff1)



### Expected result AFTER applying this Pull Request
![image](https://github.com/joomla/joomla-cms/assets/11453784/a2221bb2-5574-49e3-ba1b-b3ec3fdaea86)



### Link to documentations
Please select:
